hwdetect/hwdetect.sh

```diff
@@ -2,7 +2,10 @@
 
 function find_i2c()
 {
-	echo $(ls /dev | grep i2c)
+	mapfile -t i2c_array < <(ls /dev | grep i2c)
+        for i in ${i2c_array[@]}; do
+                echo $i
+        done
```

Comment:
I decided that it could be processed later, but I had no opportunity to check the workflow of this part of the script on my laptop. Probably I`ll check it later on the board.